cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst

```diff
@@ -32,25 +32,49 @@ compliance and governance.
 Metadata
 ========
 Metadata |---| consisting of **properties** (a list of key-value pairs) or **tags** (a
-list of keys) |---| can be used to annotate datasets, streams, programs, and applications.
+list of keys) |---| can be used to annotate artifacts, applications, programs, datasets,
+streams, and views.
 
 Using the CDAP :ref:`Metadata HTTP RESTful API <http-restful-api-metadata>`, you can set,
-retrieve, and delete the metadata annotations of applications, datasets, streams, and
-programs in CDAP.
+retrieve, and delete the metadata annotations of artifacts, applications, programs, datasets,
+streams, and views in CDAP.
 
 Metadata keys, values, and tags must conform to the CDAP :ref:`supported characters 
 <supported-characters>`, and are limited to 50 characters in length. The entire metadata
 object associated with a single entity is limited to 10K bytes in size.
 
+Discovery
+=========
 Metadata can be used to tag different CDAP components so that they are easily identifiable
-and managed. You can tag a datasets as *experimental* or an application as *production*.
+and managed. This helps in discovering CDAP components.
 
-Metadata can be **searched**, either to find entities:
+For example, you can tag a dataset as *experimental* or an application as *production*. These
+entities can then be discovered by using the annotated metadata in **search queries**. Using search,
+users can discover entities:
 
 - that have a particular **value** for *any key* in their properties;
 - that have a particular **key** with a particular *value* in their properties; or
 - that have a particular **tag**.
 
+System Metadata
+===============
+While CDAP allows users to tag entities with metadata properties and tags, it also
+automatically tags entities with system metadata. Hence, entities in CDAP have are annotated
+with some properties and tags by default. These default properties and tags can be retrieved
+using the CDAP :ref:`Metadata HTTP RESTful API <http-restful-api-metadata>` by setting the
+``scope`` query parameter to **system**. These default annotations help users to discover CDAP
+entities easily using the Metadata Search API, even if they have not explicitly annotated these
```

Comment:
`These default annotations help users to discover CDAP entities easily using the Metadata Search API, even if they have not explicitly annotated these entities.`

There is some redundancy in this sentence. I think it can be simplified to the below without loss of meaning -
`These default annotations can be used to discover CDAP entities using the Metadata Search API.`